pkg/apis/eventing/v1alpha1/subscribable_channelable_validation.go

```diff
@@ -59,20 +59,7 @@ func isSubscribableEmpty(f corev1.ObjectReference) bool {
 // - APIVersion == 'eventing.knative.dev/v1alpha1'
 // - Name       == not empty
 func isValidSubscribable(f corev1.ObjectReference) *apis.FieldError {
```

Comment:
We removed the subscribable interface recently. Hence, we can remove this method and the one above `isSubscribableEmpty`

Reply:
and an associated test here https://github.com/knative/eventing/blob/870182bf426774607b8447e90dc1ac17e268bb17/pkg/apis/eventing/v1alpha1/subscribable_channelable_validation_test.go#L101

Reply:
Removing them, thanks!